File: Bai_19/main.py
# Open the input and output files
with open('MINK.INP', 'r') as inp, open('MINK.OUT', 'w') as out:
    N = int(inp.readline().strip())  # Read the number of elements
    k = int(inp.readline().strip())  # Read the value of k
    arr = list(map(int, inp.readline().split()))  # Read the array elements

    arr.sort()  # Sort the array in ascending order

    out.write(str(arr[k - 1]))  # Write the k-th smallest element to the output file

File: Bai_19/main.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main()
{
    freopen("MINK.INP", "r", stdin);  // Redirects standard input to MINK.INP
    freopen("MINK.OUT", "w", stdout); // Redirects standard output to MINK.OUT

    int N, k;
    cin >> N >> k;

    vector<int> arr(N);
    for (int i = 0; i < N; ++i)
    {
        cin >> arr[i];
    }

    sort(arr.begin(), arr.end()); // Sort the array in ascending order

    cout << arr[k - 1]; // Output the k-th smallest element

    return 0;
}